src/main/java/gift/entity/OptionEntity.java

```diff
@@ -60,6 +60,9 @@ public void setProductEntity(ProductEntity productEntity) {
         this.productEntity = productEntity;
     }
 
+    public void subtract(Long quantity) {
+        this.quantity-=quantity;
+    }
```

Comment:
```suggestion
    public void subtractQuantity(int amountToSubtract) { // input 변수명 수정
        // amountToSubtract 가 음수인지 확인
        this.quantity -= quantity;
    }
```
변수명 변경, 예외 처리를 추가하는게 어떨까요?